src/main/java/com/internet/shop/service/impl/UserServiceImpl.java

```diff
@@ -0,0 +1,46 @@
+package com.internet.shop.service.impl;
+
+import com.internet.shop.dao.UserDao;
+import com.internet.shop.lib.Inject;
+import com.internet.shop.lib.Service;
+import com.internet.shop.model.ShoppingCart;
+import com.internet.shop.model.User;
+import com.internet.shop.service.ShoppingCartService;
+import com.internet.shop.service.UserService;
+import java.util.List;
+
+@Service
+public class UserServiceImpl implements UserService {
+    @Inject
+    private UserDao userDao;
+    @Inject
+    private ShoppingCartService shoppingCartService;
+
+    @Override
+    public User create(User user) {
+        userDao.create(user);
+        ShoppingCart shoppingCart = new ShoppingCart(user.getId());
```

Comment:
I'm not sure if we need to immediately assign the cart to the user, probably now there will be less code in main, but how will we do this next?